src/plugins/thirdParty/Python/CMakeLists.txt

```diff
@@ -79,6 +79,12 @@ endif()
 
 set(EXTERNAL_BINARIES_DIR ${FULL_LOCAL_EXTERNAL_PACKAGE_DIR}/${LIBRARY_DIR})
 
+# Make sure the directory exists
+
+file(MAKE_DIRECTORY ${EXTERNAL_BINARIES_DIR})
```

Comment:
Why do we need to do this? I mean, we don't do need anywhere else, so...?

Reply:
Hmm, historically it was happening as part of setting up `sqlite3` so I moved where it was done. But you are right, it's not needed...